common/src/main/java/tech/pegasys/web3signer/common/JacksonSerializers.java

```diff
@@ -31,9 +31,12 @@ public static class Base64Deserialiser extends JsonDeserializer<Bytes> {
     @Override
     public Bytes deserialize(final JsonParser p, final DeserializationContext ctxt) {
       try {
-        return Bytes.fromBase64String(p.getValueAsString());
+        final String valueAsString = p.getValueAsString();
+        return Bytes.fromBase64String(valueAsString);
```

Comment:
There is no point in extracting valueAsString as variable here. Also, we may not really want to print or log the invalid value at first place.